tools/export-docu-to-website.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright (c) 2020 by ApexAI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+WORKSPACE=$(git rev-parse --show-toplevel)
+WEBREPO="git@github.com:eclipse-iceoryx/iceoryx-web.git"
+VERSION=$1
+
+cd $WORKSPACE
+
+# Generate doxygen, replace with CMake commands later
+cd  $WORKSPACE/iceoryx_utils/doc/
+doxygen doxyfile-utils
+cd  $WORKSPACE/iceoryx_posh/doc/
+doxygen doxyfile-posh
+
+# Generate markdown from doxygen
+cd $WORKSPACE
+mkdir -p $WORKSPACE/doc/website/API-reference/utils
+doxybook2 --input $WORKSPACE/iceoryx_utils/doc/xml/ --output $WORKSPACE/doc/website/API-reference/utils
+mkdir -p $WORKSPACE/doc/website/API-reference/posh
+doxybook2 --input $WORKSPACE/iceoryx_posh/doc/xml/ --output $WORKSPACE/doc/website/API-reference/posh
```

Comment:
You can replace line 26 - 36 with:
```bash
# Generate doxygen, replace with CMake commands later
$WORKSPACE/tools/iceoryx_build_test.sh clean doc


# Generate markdown from doxygen
for cmp in $COMPONENTS; do
    mkdir -p $WORKSPACE/doc/website/API-reference/"$cmp"
    doxybook2 --input $WORKSPACE/build/doc/"$cmp"/xml --output $WORKSPACE/doc/website/API-reference/"$cmp"
done
```

Reply:
and on top of that
```bash
COMPONENTS="iceoryx_posh iceoryx_utils iceoryx_introspection iceoryx_binding_c iceoryx_component iceoryx_dds" 
```

Reply:
Thanks for your remark, good idea. However, I think we wanna control the name that appears on iceoryx.io:

* iceoryx_posh -> Posh
* iceoryx_utils -> Utils (or e.g safe building blocks in future)
* iceoryx_introspection -> Introspection
* iceoryx_binding_c -> C-binding
* iceoryx_component -> IMHO not necessary
* iceoryx_dds -> DDS gateway

So looping isn't possible in that case, as the folder name is picked up by MkDocs.